src/components/auth/SignUpInline.tsx

```diff
@@ -117,7 +118,7 @@ export default function SignUpInline(props: SignUpInlineProps) {
           />
         </div>
 
-        <span className="w-full text-sm text-gray-50 sm:text-left">
+        <span className={`w-full text-sm ${props.textColor ?? 'text-gray-50'} sm:text-left`}>
```

Comment:
This is [not recommended by TailwindCSS](https://tailwindcss.com/docs/content-configuration#dynamic-class-names), colors can be conditional, but if passing colors as prop they should be defined (as string, string array or string object), so dev cannot enter a random color.